### PR TITLE
Feature/order detail view

### PR DESCRIPTION
**Pull request by exmokie**

### CODE REVIEW

File: main/main_views/order_detail.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from rest_framework import generics
+from main.models import Order
+from main.serializers import OrderDetailSerializer
+
+
+# TODO: Change to Retrieve API View
+class OrderDetail(generics.RetrieveUpdateDestroyAPIView):
```
Comment (mackeyja92) [inline main/main_views/order_detail.py]: Please make it a RetreiveAPIView and change the serializer to OrderSerializer. We don't need an order detail serializer.

File: main/serializers.py [hunk 2]
```diff
@@ -18,6 +18,9 @@ class OptionSerializer(serializers.ModelSerializer):
     class Meta:
         model = Option
 
+class OrderDetailSerializer(serializers.ModelSerializer):
```
Comment (jared-mackey) [inline main/serializers.py:21]: The OrderDetailSerializer as it is redundant. Please remove and use OrderSerializer instead.

File: main/main_views/order_detail.py (hunk 1 above)
Comment (jared-mackey) [inline main/main_views/order_detail.py:7]: Please change to RetrieveAPIView as we don't want the ability to change and order from a non-admin type view. Also, please add authentication class and permission class to the view.

Comment (exmokie) [inline main/main_views/order_detail.py:7]: You change it dude. I'm going to bed.

**Comment by cauffe** [comment]
merging for now, it's good functionality and can be optimized / refactored in another pull.

**Pull request merged**